src/Symfony/Bundle/FrameworkBundle/Command/ConfigDumpReferenceCommand.php

```diff
@@ -41,13 +41,16 @@ class ConfigDumpReferenceCommand extends AbstractConfigCommand
 {
     protected function configure(): void
     {
+        $commentedHelpFormats = array_map(static fn (string $format): string => sprintf('<comment>%s</comment>', $format), $this->getAvailableFormatOptions());
+        $helpFormats = implode('", "', $commentedHelpFormats);
+
         $this
             ->setDefinition([
                 new InputArgument('name', InputArgument::OPTIONAL, 'The Bundle name or the extension alias'),
                 new InputArgument('path', InputArgument::OPTIONAL, 'The configuration option path'),
-                new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (yaml or xml)', 'yaml'),
+                new InputOption('format', null, InputOption::VALUE_REQUIRED, sprintf('The output format ("%s")', implode('", "', $this->getAvailableFormatOptions())), class_exists(Yaml::class) ? 'yaml' : 'json'),
```

Comment:
You change the default value when Yaml is not available? Smart 👍🏻

Reply:
I'm not so sure. I would prefer to have a default that is "stable". In any case, it's not related to this PR, so let's revert and discuss the default change elsewhere.

Reply:
I agree, it's not related to this PR.

(FYI ConfigDebugCommand has this behavior)